src/ui/ResultLink/ResultLink.ts

```diff
@@ -303,13 +329,19 @@ export class ResultLink extends Component {
     return (this.options.openQuickview || this.isUriThatMustBeOpenedInQuickview()) && QueryUtils.hasHTMLVersion(this.result);
   }
 
-  private parseHrefTemplate(): string {
-    return this.options.hrefTemplate.replace(/\$\{(.*?)\}/g, (value: string) => {
-      let key = value.substring(2, value.length - 1);
+  private parseStringTemplate(template: string): string {
+    if (!template) {
+      return "";
+    }
+    return template.replace(/\$\{(.*?)\}/g, (value: string) => {
+      let key = value.substring(2, value.length-1);
       let newValue = this.readFromObject(this.result, key);
       if (!newValue) {
         newValue = this.readFromObject(window, key);
       }
+      if(!newValue) {
+        this.logger.warn(`${key} is undefined for this result, please verify your syntax.`);
```

Comment:
In the warning object sent by the logger, you have the `result` object
`[1]ResultLink.result`